### PR TITLE
Add tutoring configs

### PR DESCRIPTION
**Pull request by webDevAdvocate**

### CODE REVIEW

File: packages/eslint-config-brainly-react-typescript/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: [
+    require.resolve('./rules/main.js'),
+    require.resolve('./rules/prettier.js'),
+    require.resolve('./rules/css-modules.js'),
+    require.resolve('./rules/jest.js'),
+    require.resolve('./rules/graphql.js'),
```
Comment (matikrk) [inline packages/eslint-config-brainly-react-typescript/index.js:7]: I would exclude this extend - for working graphql you need to have the correct schema file. And sometimes you don't have graphql at all :)

Comment (webDevAdvocate) [reply packages/eslint-config-brainly-react-typescript/index.js]: done

**Comment by webDevAdvocate** [comment]
> Maybe create also 2nd eslint config `base-typescript` ??

I was thinking about base-typescript config which will be extended by react-typescript and node-typescript but maybe we do it in task when node config will be created?

**Comment by matikrk** [comment]
> > Maybe create also 2nd eslint config `base-typescript` ??
> 
> I was thinking about base-typescript config which will be extended by react-typescript and node-typescript but maybe we do it in task when node config will be created?

We could use them in lambdas

**Pull request merged**